src/twisted/mail/smtp.py

```diff
@@ -1896,7 +1897,7 @@ def __init__(self, fromEmail, toEmail, file, deferred, retries=5,
                 if not isinstance(_email, bytes):
                     _email = _email.encode('ascii')
 
-                toEmailFinal.append(email)
+                toEmailFinal.append(_email)
```

Comment:
Maybe it's not a good idea to have variables named `email` and `_email` in the same function?